schema/base/land_use.yaml

```diff
@@ -123,11 +131,17 @@ properties:
           - recreation_ground
           - religious
           - residential
+          - resort
           - retail
           - rough
           - salt_pond
           - school
           - schoolyard
+          - ski_jump
+          - skitour
+          - sled
+          - sleigh
```

Comment:
What is the difference between `sled` and `sleigh`?

Reply:
I'm not 100% sure: Sled has a wiki page: https://taginfo.openstreetmap.org/tags/piste%3Atype=sled#overview

`sleigh` is present, but less used? @BoZivk?

Reply:
They are both supported based on this: https://wiki.openstreetmap.org/wiki/Key:piste:type

Reply:
Thanks @BoZivk, sounds good to me.